client/src/components/Nav/Nav.jsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom';
+import styles from './Nav.scss';
+
+export function Nav() {
+  return (
+    <div className={styles.root}>
+      <Link to="/newest">newest</Link>
+      &nbsp;|&nbsp;
```

Comment:
In my opinion, using `border` and `padding` is a better approach for this...

Reply:
because there is no any ui description, this approach is copy from hacknews